pkg/operator/controller/names.go

```diff
@@ -46,6 +52,14 @@ func IngressClusterOperatorName() types.NamespacedName {
 	}
 }
 
+// IngressClusterConfigName returns the namespaced name of the ingress.config.openshift.io
+// resource for the operator.
+func IngressClusterConfigName() types.NamespacedName {
```

Comment:
over-abstraction

Reply:
I'm hesitant to break from the pattern established by the authors of the operator. @sgreene570 could you weigh in here?

Reply:
I would prefer to keep things consistent with the existing operator code but if you feel strongly about moving / modifying / removing this function I won't object.